### PR TITLE
Updates class/struct formatting rules to include empty space.

### PR DESCRIPTION
**Pull request by RCacheaux**
I'm working on a book and this PR reflects the changes made in formatting in order to include a space below the type signature line.

### CODE REVIEW

**Comment by rayfix** [comment]
What is the motivation for this change @crispy8888 ?  Is it because it looks better on the site?

File: README.markdown [hunk 1]
```diff
@@ -726,6 +735,7 @@ Use access control as the leading property specifier. The only things that shoul
 private let message = "Great Scott!"
 
 class TimeMachine {  
+
```
Comment (rayfix) [inline README.markdown]: This space feels unnecessary for clarity to me.

Comment (rcritz) [reply README.markdown]: I also think it's unnecessary and not consistent with the way we edit tutorials and other books.

**Comment by rcritz** [comment]
Actually, we're not going to make this change. There should be no blank lines following a open brace or before a closing brace. I've added a bullet under **Spacing** to make this clear.

**Pull request closed without merging**